libr/core/panels.c

```diff
@@ -3714,6 +3715,18 @@ R_API int r_core_visual_panels(RCore *core, RPanels *panels) {
 			cur->model->directionCb (core, (int)DOWN);
 		}
 		break;
+	case 'H':
+		r_cons_switchbuf (false);
+		for (i = 0; i < PANEL_CONFIG_PAGE; i++) {
+			cur->model->directionCb (core, (int)LEFT);
+		}
+		break;
+	case 'L':
+		r_cons_switchbuf (false);
+		for (i = 0; i < PANEL_CONFIG_PAGE; i++) {
```

Comment:
uh? isn this PAGE thing a constant size? the width != height, and also maybe it should scroll half page instead of full page (same with JK).

Reply:
this is a constant of 10. maybe worth considering frame->w /2; or frame->h /2 ?